applications/GeoMechanicsApplication/custom_conditions/T_normal_flux_condition.cpp

```diff
@@ -81,8 +81,7 @@ void GeoTNormalFluxCondition<TDim, TNumNodes>::CalculateRHS(Vector& rRightHandSi
 
         // Contributions to the right hand side
         auto normal_flux_on_DOF = normal_flux_on_integration_point * N * weighting_integration_coefficient;
-        GeoElementUtilities::AssemblePBlockVector<0, TNumNodes>(rRightHandSideVector,
-                                                                normal_flux_on_DOF);
+        rRightHandSideVector += normal_flux_on_DOF;
```

Comment:
or
rRightHandSideVector += normal_flux_on_integration_point * N * weighting_integration_coefficient;

Reply:
Done. For clarity, I have enclosed the expression that follows `+=` in parentheses.